src/app/shared/components/input/Input.tsx

```diff
@@ -1,10 +1,8 @@
-import React, { Component } from 'react';
+import React from 'react';
 import './input.scss';
 
-class Input extends Component {
-  render() {
-    return <p className="input">Input</p>;
-  }
-}
+export const Input = ({ type = 'text', placeholder = '' }) => {
```

Comment:
I think you don't need to use an arrow function here if you are not going to use something from a higher scope

Reply:
Ok, that make sense.